test/javascript/packs/utils/calendar.test.js

```diff
@@ -24,4 +25,48 @@ describe("calendar", () => {
       });
     });
   });
+
+  describe("prev", () => {
+    const base_date = moment('2022-08-01').utc()
+    it('returns previous month in same day', () => {
+      const prev_month = prev(base_date)
+      expect(prev_month.format('YYYY-MM-DD')).toBe('2022-07-01')
+    });
+  });
+
+  describe("next", () => {
+    const base_date = moment('2022-08-01').utc()
+    it('returns next month in same day', () => {
+      const next_month = next(base_date)
+      expect(next_month.format('YYYY-MM-DD')).toBe('2022-09-01')
+    });
+  });
+
+  describe("innerRange", () => {
+    const currentDate = moment('2022-08-15').utc()
+    describe('when the base date month is the same as current date', () => {
+      const baseDate = moment('2022-08-01').utc()
+      it('returns a range from base date until current date', () => {
+        const range = innerRange(baseDate, currentDate)
+        expect(range[0].format('YYYY-MM-DD')).toBe('2022-08-01')
+        expect(range[1].format('YYYY-MM-DD')).toBe('2022-08-15')
+      });
+    });
+
+    describe("when the base date is one or more month(s) older than current date", () => {
+      const baseDate = moment('2022-06-01').utc()
+      it('returns a range from base date until one day before next month', () => {
+        const range = innerRange(baseDate, currentDate)
+        expect(range[0].format('YYYY-MM-DD')).toBe('2022-06-01')
+        expect(range[1].format('YYYY-MM-DD')).toBe('2022-06-30')
+      });
+    })
+  });
+
+  describe('startDate', () => {
```

Comment:
When we run this test, there is a warning in the console. Shouldn't it be fixed?

![image](https://user-images.githubusercontent.com/54550926/186961508-bc40d5f9-e202-400b-a9f7-887b0b32d1ec.png)